### PR TITLE
feat(core): migration deploy

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add CLI parameter `--deploy-migration` to run migration files.

## Testing

TBD

### CODE REVIEW

File: packages/core/src/database/migration.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  await pool.query(sql`${raw(query)}`);
+
+  return true;
+};
+
+type MigrationScript = { up: (pool: DatabasePool) => Promise<void> };
+
+const getMigrationFiles = async () => {
+  const directory = await readdir(migrationFilesDirectory);
+  const files = directory.filter((file) => file.endsWith('.js'));
```
Comment (simeng-li) [inline packages/core/src/database/migration.ts:39]: will we keep other files under this migration path?

File: packages/core/src/index.ts [hunk 2]
```diff
@@ -10,13 +10,21 @@ configDotEnv();
...
 /* eslint-enable import/first */
 
 (async () => {
   try {
     await envSet.load();
+
+    if (envSet.isDeployMigration) {
+      await deployMigration(envSet.pool);
+
+      return;
```
[the remark this answers is not in the document]
Comment (wangsijie) [reply packages/core/src/index.ts:25]: It is designed to be a single job

File: packages/core/src/database/migration.ts (hunk 1 above)
Comment (wangsijie) [reply packages/core/src/database/migration.ts:39]: No, only migration files.

Comment (wangsijie) [reply packages/core/src/database/migration.ts:39]: Though, we can add some file name check.

**Pull request closed without merging**